scripts/generate_coverage/merge_cobertura.py

```
@@ -0,0 +1,42 @@
+# dependencies = ["plumbum", "typer"]
+# ///
+"""Merge Cobertura XML files from Rust and Python coverage runs."""
+
+from pathlib import Path
+
+import typer
+from plumbum.cmd import uvx
+from plumbum.commands.processes import ProcessExecutionError
+
+RUST_FILE_OPT = typer.Option(
+    ..., envvar="RUST_FILE", exists=True, file_okay=True, dir_okay=False
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)